client/me/purchases/non-primary-domain-dialog/index.jsx

```diff
@@ -77,6 +77,23 @@ class NonPrimaryDomainDialog extends Component {
 								},
 							}
 						) }
+						{ hasSetupAds &&
+							translate(
+								'{{br/}}{{br/}}You will also be ineligible for the WordAds program. Visit {{a}}our FAQ{{/a}} to learn more.{{br/}}{{br/}}',
+								{
+									components: {
+										a: (
+											<a
+												href="https://wordads.co/faq/#eligibility-for-wordads"
+												target="_blank"
+												rel="noopener noreferrer"
+											/>
+										),
+										br: <br />,
+									},
+								}
+							) }
+						{ translate( ' Would you still like to downgrade your plan?' ) }
```

Comment:
Translating strings that start with a space for layout purposes is not a great idea for localization. What if you just put a space between this string and the previous ones instead?